packages/manager/modules/pci/src/projects/project/notebooks/onboarding/onboarding.constants.js

```diff
@@ -1,26 +1,26 @@
 export const GUIDES = [
   {
     id: 'documentation',
-    link: 'https://docs.ovh.com/gb/en/publiccloud/ai/notebooks/',
+    link: 'https://docs.ovh.com/gb/en/publiccloud/ai/',
   },
   {
     id: 'definition',
-    link: 'https://docs.ovh.com/gb/en/publiccloud/ai/notebooks/definition/',
+    link: 'https://docs.ovh.com/gb/en/publiccloud/ai/ai-comparative-tables/',
   },
   {
     id: 'start',
     link:
-      'https://docs.ovh.com/gb/en/publiccloud/ai/notebooks/getting-started-cli/',
+      'https://docs.ovh.com/gb/en/publiccloud/ai/notebooks/definition/',
```

Comment:
Both line 12 and 17 are not well formatted.

```sh
$ yarn eslint packages/manager/modules/pci/src/projects/project/notebooks/onboarding/onboarding.constants.js
yarn run v1.22.17
$ /home/ovh/manager/node_modules/.bin/eslint packages/manager/modules/pci/src/projects/project/notebooks/onboarding/onboarding.constants.js

/home/ovh/manager/packages/manager/modules/pci/src/projects/project/notebooks/onboarding/onboarding.constants.js
  12:10  error  Delete `⏎·····`  prettier/prettier
  17:10  error  Delete `⏎·····`  prettier/prettier

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.

error Command failed with exit code 1.
```

Reply:
Do you have public documentation about this ? I see nothing in Contributing neither How To.
Thanks !

Reply:
Once the project is installed locally, you'll get some tooling that will help you with the linting.

```sh
yarn # or yarn install
git add .
git commit -s # this will triggers linting tasks via git-hooks.
```